app/services/photo.js

```diff
@@ -1,6 +1,14 @@
 const axios = require('axios');
+const logger = require('../logger');
+const errors = require('../errors');
 
 exports.photos = () =>
-  axios.get('https://jsonplaceholder.typicode.com/photos').then(response => {
-    console.log(response);
-  });
+  axios
+    .get('https://jsonplaceholder.typicode.com/photos')
+    .then(response => {
+      logger.info(`Received photos: ${JSON.stringify(response)}`);
+      return response;
+    })
+    .catch(() => {
+      throw errors.external_api_error('Error in external API');
+    });
```

Comment:
Move this to the albums service. We usually only have one service file per external service, and here you are just getting the photos of the albums.